src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrRegistrations.cs

```diff
@@ -65,6 +65,8 @@ private static void RegisterCommonServices(IServiceCollection services, string g
         services.TryAddTransient<IGitRepoClonerFactory, GitRepoClonerFactory>();
         services.TryAddTransient<VmrCloakedFileScanner>();
         services.TryAddTransient<VmrBinaryFileScanner>();
+        services.AddHttpClient();
```

Comment:
We could also have a named Http client that's configured here with things like base address (https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-7.0) What do you think @premun ?

Reply:
Yes, good point. Named HTTP clients bring a lot of clarity and help you find where the client is defined.

Reply:
we can also use `HttpRequestManager` from `DarcLib` for sending the request, what do you think

Reply:
What @dkurepa meant is this: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-7.0#named-clients

You could name it `GraphQLclient` and ask for it directly.

I like this `HttpRequestManager` though as it does retries and so on.

Reply:
yes, I meant use the manager as well, not instead of what you suggested

Reply:
Yeah, let's use it, it looks nice